projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -380,7 +380,11 @@ if (environment.cpq) {
         a11yLinkBtnsToTertiaryBtns: true,
         a11yRepeatedPageTitleFix: true,
         a11yDeliveryModeRadiogroup: true,
-        a11yNgSelectOptionsCount: true,
+        /**
+         * Defaults to false cause ng-select options ariaLabels are working as expected
+         * since Spartacus 2211.33
```

Comment:
Lets also reference the issue CX-9005 here and in `feature-toggles.ts` so anybody curious can reference the discussion about this flag.